core/src/main/kotlin/ClientResources.kt

```diff
@@ -1,18 +1,18 @@
 package dev.kord.core
 
-import dev.kord.common.entity.Snowflake
+import dev.kord.core.builder.kord.Shards
 import dev.kord.core.supplier.EntitySupplyStrategy
 import dev.kord.gateway.Intents
-import io.ktor.client.HttpClient
+import io.ktor.client.*
 
 class ClientResources(
     val token: String,
-    val shardCount: Int,
+    val shards: Shards,
     val httpClient: HttpClient,
     val defaultStrategy: EntitySupplyStrategy<*>,
     val intents: Intents,
 ) {
     override fun toString(): String {
-        return "ClientResources(shardCount=$shardCount, httpClient=$httpClient, defaultStrategy=$defaultStrategy, intents=$intents)"
+        return "ClientResources(shardCount=$shards, httpClient=$httpClient, defaultStrategy=$defaultStrategy, intents=$intents)"
```

Comment:
```suggestion
        return "ClientResources(shards=$shards, httpClient=$httpClient, defaultStrategy=$defaultStrategy, intents=$intents)"
```

Reply:
There is no need for this, sharding is a data class.

Reply:
mb, I mis-understood the point of the change